torch/_torch_docs.py

```diff
@@ -145,21 +145,21 @@ def merge_dicts(*dicts):
            r"""
 .. function:: add(input, other, out=None)
 
-Adds the scalar :attr:`other` to each element of the input :attr:`input`
-and returns a new resulting tensor.
+Each element of the tensor ``other`` is added to each element of the
+tensor ``input``.
 
 .. math::
     \text{{out}} = \text{{input}} + \text{{other}}
 
-If :attr:`input` is of type FloatTensor or DoubleTensor, :attr:`other` must be
-a real number, otherwise it should be an integer.
-
-Args:
-    {input}
-    value (Number): the number to be added to each element of :attr:`input`
+The shapes of :attr:`input` and :attr:`other` must be
+:ref:`broadcastable <broadcasting-semantics>`.
 
-Keyword arguments:
-    {out}
+The :class:`torch.dtype` of the result tensor is the result of calling
+:meth:`torch.result_type` with the :attr:`input` and :attr:`other`
```

Comment:
As a user, I don't want to click through and understand `torch.result_type` before using `torch.add`. Maybe say something like "if the input and other have different dtype, the output has a different dtype according to type promotion rules"

Reply:
reworded